frontend/src/Store/Actions/seriesActions.js

```diff
@@ -261,6 +273,31 @@ export const filterBuilderProps = [
       return tagList.sort(sortByName);
     }
   },
+  {
+    name: 'releaseGroups',
+    label: 'Release Groups',
+    type: filterBuilderTypes.ARRAY,
+    optionsSelector: function(items) {
+      const releaseGroupsList = items.reduce((acc, series) => {
```

Comment:
I think no suggestions is better, otherwise this is going to be a lot of processing for large lists of series, people can freeform it.